pkg/utils/transform/transform_test.go

```diff
@@ -55,6 +59,13 @@ func TestReplaceKind(t *testing.T) {
 	})
 }
 
+func assertKind(t *testing.T, u unstructured.Unstructured, kind string) {
+	t.Helper()
+	if k := u.GetKind(); k != kind {
+		t.Errorf("expected kind %s, got kind %s", kind, k)
+	}
+}
+
```

Comment:
nit: move this assertFunc to the end of the file with other assertFunc

Reply:
we can. The only reason I moved those function close to the caller because often time its easy find it.